src/d2d1.rs

```diff
@@ -0,0 +1,1405 @@
+// Copyright © 2015, Connor Hilarides
+// Licensed under the MIT License <LICENSE.md>
+//! Mappings for the contents of d2d1.h, d2dbasetypes.h, and dcommon.h
+
+//-------------------------------------------------------------------------------------------------
+// D2D1_DEFAULT_FLATTENING_TOLERANCE
+//-------------------------------------------------------------------------------------------------

```

Comment:
Heh, I was just trying to provide some separation between the walls of constants, should I remove them?